pkgs/development/python-modules/astropy-healpix/default.nix

```diff
@@ -0,0 +1,27 @@
+{ pkgs
+, buildPythonPackage
+, fetchPypi
+, numpy
+, astropy
+}:
+
+buildPythonPackage rec {
+  pname = "astropy-healpix";
+  version = "0.4";
+
+  doCheck = false; # the tests requires several pytest plugins that are not in nixpkgs
```

Comment:
Which ones?

Reply:
`pytest-astropy` and its dependencies (`pytest-doctestplus`, `pytest-remotedata`, `pytest-openfiles` and `pytest-arraydiff`).